src/services/liking.service.js

```diff
@@ -13,6 +13,19 @@ const LikingService = {
     }
     like.update({ isLiked: !like.isLiked });
     return sendResult(res, 200, ` The accommodation like  status updated to ${like.isLiked}`, like);
+  },
+
+  async getLikes(req, res) {
+    const { accommodationId } = req;
+    const countLikes = await db.Likings
+      .findAndCountAll({ where: { accommodationId, isLiked: true }, raw: false });
+    const countUnlikes = await db.Likings
```

Comment:
Good job here. But do we really need to calculate the number of `unlikes`? All we need is the number of likes, an aggregate. cc @alexismajya